BinariesCheck.py

```diff
@@ -51,6 +57,18 @@ class BinaryInfo:
     setuid_call_regex = create_regexp_call(['setresuid', 'seteuid', 'setuid'])
     setgroups_call_regex = create_regexp_call(['initgroups', 'setgroups'])
     chroot_call_regex = create_regexp_call('chroot')
+
+    forbidden_functions = Config.getOption("WarnOnFunction")
+    if forbidden_functions:
+        for name, func in forbidden_functions.items():
```

Comment:
I've used `items()` instead of `iteritems()` as the former is Python 3 compatible